prom2teams/app/configuration.py

```diff
@@ -107,6 +108,8 @@ def config_app(application):
             application.config['LOG_FILE_PATH'] = command_line_args.logfilepath
         if command_line_args.templatepath:
             application.config['TEMPLATE_PATH'] = command_line_args.templatepath
+        if command_line_args.templatepath:
+            application.config['GROUP_ALERTS'] = command_line_args.groupalerts == "TRUE"
```

Comment:
To avoid change the behaviour of users who update the version i'ld set to FALSE by default